test/files/SPS/sps007/pass-sps007-scenario02-annotation_hosted_element.ifc

```diff
@@ -0,0 +1,144 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition[Ifc4x3NotAssigned]'),'2;1');
+FILE_NAME('testfile.ifc','2022-09-16T10:35:07',('John Smith'),('buildingSMART Int.'),'MyToolboxName Version 1.0','Acme Inc. - TNT BIM 2025 - 25.1.0.0','');
+FILE_SCHEMA(('IFC4X3_ADD2'));
+ENDSEC;
+DATA;
+#1=IFCPROJECT('2LJesmETv6W9jWXnI$q1Aj',#2,'sOhp2Q','aVY0hE',$,$,$,(#12),#7);
+#2=IFCOWNERHISTORY(#3,#6,$,.NOCHANGE.,$,$,$,1736381325);
+#3=IFCPERSONANDORGANIZATION(#4,#5,$);
+#4=IFCPERSON($,'nuvnTe','2i2zIDMqly',$,$,$,$,$);
+#5=IFCORGANIZATION($,'Ejm7vQJXZ','h5GCeytk',$,$);
+#6=IFCAPPLICATION(#5,'15.0C1r [DEVELOP] Pre RC3','QTOQcV','6WjUdPqnCj');
+#7=IFCUNITASSIGNMENT((#8,#9,#10,#11));
+#8=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#9=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#10=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#11=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#12=IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,1.E-06,#14,#18);
+#13=IFCLOCALPLACEMENT($,#14);
+#14=IFCAXIS2PLACEMENT3D(#15,#16,#17);
+#15=IFCCARTESIANPOINT((0.,0.,0.));
+#16=IFCDIRECTION((0.,0.,1.));
+#17=IFCDIRECTION((1.,0.,0.));
+#18=IFCDIRECTION((0.,1.));
+#19=IFCPROJECTEDCRS('n7rJmd','7yPIU9FJik','EPSG:7844','EPSG:5711','UTM','56',$);
+#20=IFCRIGIDOPERATION(#12,#19,0.,0.,0.);
+#21=IFCRELDECLARES('1wYK6WxCrEVuvIYx98krR$',#2,'vofFGYo','nDwErbk4bU',#1,(#22));
+#22=IFCACTOR('2mtFEuUsjAZvsYlSxLubX3',$,'XA5xdjh','m4g56gsA7B',$,#4);
+#23=IFCSITE('2G0ATHGhj5T8pYNNtAHW7K',#2,'qfpeGrpS','oF7Hjskd4',$,#13,$,$,.ELEMENT.,$,$,$,$,$);
+#24=IFCRELAGGREGATES('23F9xPv0L64BRir8Qemk5E',#2,'cq5acgJ','HRnXa7z',#1,(#23));
+#25=IFCFACILITY('1ImCI2aTD1DQ6ZcsL4wZU1',#2,'JlsLnx4','7oXx5GP7',$,#13,$,$,.ELEMENT.);
+#26=IFCRELAGGREGATES('2AnbcmI2r3HwhOaiFhHKMQ',#2,'JXZia1evi','VmSXvwm',#23,(#25));
+#27=IFCRELCONTAINEDINSPATIALSTRUCTURE('0y1pwwXIn9pwEP14uHse4P',#2,'Pa5r3Notr','vinelT',(#28),#25);
```

Comment:
So this is correct, but I think it would be good to have the 'reverse' as another fail file. Where we contain the 'nested' rather than the 'nestee'.

#27=IFCRELCONTAINEDINSPATIALSTRUCTURE('0y1pwwXIn9pwEP14uHse4P',#2,'Pa5r3Notr','vinelT',(#30,#43,#56,#66),#25);

Because:

> Given A relationship IfcRelNests does not exist to IfcAnnotation from IfcAnnotation

is not very clear of  course in terms of the directionality